tr/downloads/index.md

```diff
@@ -4,8 +4,6 @@ title: "Ruby'yi İndirin"
 lang: tr
 ---
 
-{% include out-of-date.html %}
```

Comment:
aslinda bunu kaldirmak dogru mu bilemedim. turkce bir icerik ararken henuz cevrilmemisse ne oluyor? ya da o icerik sitede hic yoksa turkce mi uyari cikiyor?

Reply:
Bu uyarı sadece downloads belgesi için geçerli. Şu an o sayfa tam olarak çevrildiği için buna gerek yok. Ana sayfada unmaintained diye bir uyarı var zaten.

Reply:
İçerik yoksa (çevrilmemişse) 404 çıkıyor ama İngilizce. Onu çeviremiyoruz.